proposals/stage-0/SECURITY_REPORTING/README.md

```diff
@@ -0,0 +1,60 @@
+# My New Proposal
+>  Stage 0
+
+## Champion
+
+Marcin Hoppe (@marcin_hoppe)
+
+## Description
+
+Currently OpenJS Foundation projects do not have consistently documented standards for responsible reporting and disclosure of security vulnerabilities. This proposal aligns OpenJS Foundation projects with industry best practices in this space.
+
+See also:
+- [Security reporting in OpenJS Foundation projects](https://gist.github.com/MarcinHoppe/b13a870770522c31a8386ada48b2e40f)
+- [Guidelines for responsible reporting and disclosure of security vulnerabilities](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/security-guidelines.md)
+
+## Required Resources
+
+- Generic security policy template that projects can easily adopt
+- Optionally: vulnerability disclosure platform (VDP)
+
+## Who would be responsible?
+
+The responsibility for adoption of security reporting standards would be jointly owned by the projects and the CPC.
```

Comment:
I don't think it's the CPC responsibility. The CPC can facilitate/help it, but in the end it's for projects to implement.

Reply:
Agree that it's not up to the CPC to implement, but it should facilitate as much as possible to reach the success metrics below.
See also my comment about using a default platform for those that don't use one to make things easier.

Reply:
@mcollina @molant I agree and I changed the wording, I'd love feedback if this more accurately describes the desired roles and responsibilities.